### PR TITLE
Update glossary.md

### PR DESCRIPTION
**Pull request by mgeisler**
Expand a few random entries of the glossary. This is just to remind people that the glossary exists and that we should work on making it complete.

Part of #1466.

### CODE REVIEW

**Review by randomPoison — APPROVED** [review]
The format action found a typo that you should fix before merging this. Check the "Files changed" tab to see. Otherwise this seems fine, so I'm going to approve it now and let you merge once you've fixed that (unless the repo settings are setup to clear approvals when new commits are pushed?)

**Comment by mgeisler** [comment]
Thanks for the review!

> (unless the repo settings are setup to clear approvals when new commits are pushed?)

Nope, approvals stay for people who have write access. I find that very convenient because of how it removes mostly unnecessary round-trips.

**Pull request merged**
